app/templates/footer.html

```diff
@@ -0,0 +1,10 @@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" rel="stylesheet">
```

Comment:
I'd prefer not to import third party libs like font-awesome. So far the project is getting by fine with just vanilla JS and internal styling only. 

Could we just use a unicode character like "⬆️" or "🟢" or something? It isn't as pretty, but would work well enough imo.